PMM-9318 user management in PMM (#677)
percona/pmm-doc: docs/how-to/manage-users.md

# Manage users

This topic explains user management in PMM.

You can manage users from the [main menu](../details/interface.md#main-menu) by navigating to *Server Admin → Users* page.

![!image](../_images/PMM-manage-users.png)

## Add users

You can add a user in PMM from *User → New user* tab.

![!image](../_images/PMM-add-new-user.png)

To add a new user in PMM:

1. On the *Users* tab, click *New user*.
2. On the *Add new user* dialog box, enter the following:
    - Name

>     - email address or username (if you are an existing grafana user)

Reviewer: ```suggestion
    - email address or username (if this is an existing grafana user)
```

Reviewer: It could be that an admin is creating/updating another user.
